CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - raise on `CREATE INDEX CONCURRENTLY` https://github.com/plausible/ecto_ch/pull/121
 - switch from `DROP INDEX` to `ALTER TABLE ... DROP INDEX` syntax for indexes https://github.com/plausible/ecto_ch/pull/122
 - add support for structured `:options` in migrations https://github.com/plausible/ecto_ch/pull/116
+- add `:default_table_engine` option https://github.com/plausible/ecto_ch/pull/123
```

Comment:
`:default_table_options` instead of `:default_table_engine`

Reply:
🙈